File: Java/Basic/Max.java
import java.util.*;
import java.lang.*;
import java.io.*;
 
class Main
{
	public static void main( String[] args ) {
    	float max = 0;
		float[] data = { 1, 2, 3, 4, 5};
		// 最大値を求める
		for(int i = 0; i < data.length; i++)
			max = Math.max(max,data[i]);
		System.out.println(max);
	}
}

File: CSharp/Basic/max.cs
using System;
using System.Linq;

namespace Test
{
	class Program
	{
		static void Main(string[] args)
		{
			double[] data = { 5.1,4.2,3.1,2.2,1.4 };
			double max = data.Max();
			Console.WriteLine("最大値：" + max);
		}
	}
}
